overlays/nodejs/autoscaling/notificationType.ts

```diff
@@ -0,0 +1,19 @@
+// This file contains two categories of exports:
```

Comment:
Could you please include the standard copyright/license banner on the new file.

Reply:
I've added the copyright notice from other files in the latest commit.